src/seedu/addressbook/data/person/Address.java

```diff
@@ -56,4 +57,14 @@ public int hashCode() {
     public boolean isPrivate() {
         return isPrivate;
     }
+    
+    @Override
+    public String getPrintableString(){
+    	if (!this.isPrivate()) {
+        	return "Address: " + this.toString();
+    	}
+    	else {
+    		return ADDRESS_PRIVATE_STRING;
+    	}
+    }
```

Comment:
👍 on checking `isPrivate`. Do you think printing `This address is not printable.` is a bit too wordy?  How about just `***`?